redis.go

```diff
@@ -4,13 +4,24 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"strings"
 	"sync"
 
 	redigo "github.com/go-redis/redis/v8"
 )
 
 var ctx = context.Background()
 
+const redisPrefix = "GOIPAM"
```

Comment:
This is a breaking change for existing deployments

Reply:
could be an option to NewRedis()

Reply:
reverted as discussed. Let's see how https://github.com/metal-stack/go-ipam/pull/43 would work